tests/regression/pip_faster.py

```diff
@@ -63,10 +63,6 @@ def test_circular_dependencies():
     reason='ancient versions are not py3 compatible, even for install',
 )
 @pytest.mark.parametrize('reqs', [
-    # old setuptools and old pip
-    ['setuptools==0.6c11', 'pip==1.4.1'],
-    # old setuptools and new pip
-    ['setuptools==0.6c11', 'pip==1.5.6'],
```

Comment:
Another "fails on Python2 and not worth figuring out why" unless someone has other opinions.